CONTRIBUTING.md

```diff
@@ -1,34 +1,34 @@
 # How to Contribute
 
-We'd love to accept your patches and contributions to this project. There are
-just a few small guidelines you need to follow.
-
-## Development Principles (for Googlers)
-
-There are a few principles for developing or refactoring the service
-implementations. Read the [Development Principles
-Guide](./docs/development-principles.md).
+Thank you so much for your interest in contributing to Online Boutique.
+Before contributing, you must:
+1. sign the [Contributor License Agreement (CLA)](#contributor-license-agreement).
+2. follow [Google's Open Source Community Guidelines](https://opensource.google.com/conduct/).
+3. follow the [Contribution Process](#contribution-process).
 
 ## Contributor License Agreement
 
-Contributions to this project must be accompanied by a Contributor License
-Agreement. You (or your employer) retain the copyright to your contribution;
-this simply gives us permission to use and redistribute your contributions as
+Contributions to Online Boutique must be accompanied by a Contributor License
+Agreement (CLA). You (or your employer) retain the copyright to your contribution.
+The CLA simply gives us permission to use and redistribute your contributions as
```

Comment:
It would flow better if you either:
- Revert the removal of the `;` or
- Remove the word `simply`

Reply:
Agreed. I've removed `simply`.